schemas/dbt_project.json

```diff
@@ -352,6 +355,22 @@
             "additionalProperties": {
                 "$ref": "#/$defs/test_configs"
             }
+        },
+        "docs_config": {
+            "title": "Docs config",
+            "type": "object",
+            "description": "Configurations for the appearance of nodes in the dbt documentation.",
+            "properties": {
+                "node_color": {
+                    "type": "string",
+                    "description": "The color of the node on the DAG in the documentation"
```

Comment:
Optional extras for experts: 
- In the description, say that it can either be a CSS colour or a hex code 
- add a regex validating that if it starts with a `#` it is 3 or 6 chars (and whatever else you feel like doing to prove that it’s a hex code)